src/include/migraphx/match/gelu_tanh.hpp

```diff
@@ -39,6 +89,11 @@ auto gelu_tanh(F f)
 {
     return detail::gelu_tanh_matcher<F>{f}.matcher();
 }
+template <class F>
+auto gelu_tanh_inner(F f)
```

Comment:
I think this function is no longer used so we can just remove it.

Reply:
done